content/general/concepts/software-development-life-cycle/terms/big-bang/big-bang.md

```diff
@@ -0,0 +1,55 @@
+---
+Title: 'Big Bang'
```

Comment:
```suggestion
Title: 'Big Bang Model'
```

Reply:
We have decided to updated all of the SLDC terms to include model in the title and the file name per https://github.com/Codecademy/docs/pull/2177. Please update this entry to match.

@yangc95